actions/templates.go

```diff
@@ -137,54 +133,46 @@ func PostTemplate(c *gin.Context) {
 		return
 	}
 
-	store, err := templates.NewSesTemplateStore(keys.AccessKey, keys.SecretKey, keys.Region)
-	if err != nil {
-		logger.From(c).WithError(err).Error("Unable to create SES template store.")
-		c.JSON(http.StatusBadRequest, gin.H{
-			"message": "SES keys are incorrect.",
-		})
-		return
+	template := &entities.Template{
+		UserID:      u.ID,
+		Name:        body.Name,
```

Comment:
We'll need to check if name is unique and return an error. See the campaign implementation for Post and Put.